ground/uavobjgenerator/generators/gcs/uavobjectgeneratorgcs.cpp

```diff
@@ -480,3 +497,12 @@ bool UAVObjectGeneratorGCS::process_object(ObjectInfo* info)
 
     return true;
 }
+
+/**
+ * Escapes a raw string so it can be used in generated C/C++ source
+ * Whitespace will be gobbled
+ */
+QString UAVObjectGeneratorGCS::escape_raw_string(QString raw)
+{
+    return raw.replace("\\", "\\\\").replace("\"", "\\\"").simplified();
```

Comment:
The QString::simplified function replaces all whitespace chars with a single space, and gobbles consecutive whitespace into a single space.